cime_config/acme/allactive/config_pes.xml.cime2

```diff
@@ -454,6 +450,15 @@
     <PES_LEVEL>1r</PES_LEVEL>
 </pes>
 
+<pes OS="BGP">
```

Comment:
Notice the file has .cime2 on the end.  Here just for historical reference and not used/read.